### ISSUE TITLE
Non Fungible Link goes to Fungible Page

### ISSUE THREAD

**Issue by victor-geere** [FlowFans/flow-tools#1]
I have fixed it (and a couple of other things) here:
https://github.com/victor-geere/flow-tools

Let me know if you want a pull request.

The NFT Collections don't work so you still can't create an NFT but the link is fixed.

**Comment by caosbad** [comment]
Yeah, since the testnet upgrate , we haven't test the contract yet, PR is very welcome, we can  build flow-tools together ..

**Comment by victor-geere** [comment]
Cool. I'll cut a new branch and submit a clean PR.

**Comment by caosbad** [comment]
Looking forward it ;-}

**Comment by victor-geere** [comment]
Sorry I haven't cleaned up my branch yet. If you like you can pull this code 

`https://github.com/victor-geere/flow-tools.git`

I've fixed some links on the front page to differentiate between FT and NFT and set the cursor pointer etc.

**Comment by caosbad** [comment]
Okay
